src/components/Popover/index.tsx

```diff
@@ -78,20 +75,29 @@ export interface PopoverProps {
   show: boolean
   children: React.ReactNode
   placement?: Placement
+  hideArrow?: boolean
```

Comment:
I don't think the `?` is correct here, since there's a default argument

Reply:
TS complains if I remove the `?` since the props interface doesn't know about the default value.